tsconfig.json

```diff
@@ -1,6 +1,9 @@
 {
   "compilerOptions": { "esModuleInterop": true, "noEmit": true },
-  "references": [{ "path": "./packages/metamask-module-template" }],
+  "references": [
+    { "path": "./packages/metamask-module-template" },
+    { "path": "./packages/storybook-react" }
```

Comment:
It automatically did this with the command, should I change/remove @georgewrmarshall ?

Reply:
i'll remove in my next PR